backstop/tests/header.html

```diff
@@ -114,76 +114,79 @@ <h2>Default</h2>
 
           <div class="iui-divider"></div>
 
-          <!-- Project Button -->
-          <button
-            class="iui-button iui-borderless iui-large iui-header-button"
-            type="button"
-            aria-label="Project picker"
-            aria-expanded="false"
-            id="test-button-1"
-          >
-            <div class="iui-icon iui-header-button-icon demo-picture"></div>
-            <span class="iui-label">
-              <div>Project Alpha</div>
-              <div class="iui-description">0x0123456789</div>
-            </span>
-            <!-- Use caret-down-small.svg from @itwin/itwinui-icons package. -->
-            <svg
-              viewBox="0 0 16 16"
-              aria-hidden="true"
-              class="iui-icon"
-            >
-              <path
-                d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
-              />
-            </svg>
-          </button>
-
-          <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
-          <svg
-            viewBox="0 0 16 16"
-            aria-hidden="true"
-            class="iui-chevron"
-          >
-            <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
-          </svg>
-
-          <!-- IModel Button -->
-          <button
-            class="iui-button iui-borderless iui-active iui-large iui-header-button"
-            type="button"
-            aria-label="Model picker"
-            aria-expanded="false"
-          >
-            <!-- Use model.svg from @itwin/itwinui-icons package. -->
-            <svg
-              aria-hidden="true"
-              viewBox="0 0 16 16"
-              class="iui-icon iui-header-button-icon"
-            >
-              <path
-                d="m14 11.126-6-2.954v-6.811l-.5-.246-.5.246v6.811l-6 2.954v.559l.565.278 5.935-2.922 5.935 2.922.565-.278z"
-                opacity=".33"
-              />
-              <path
-                d="m7.5 0-7.5 3.69232v8.61536l7.5 3.69232 7.5-3.69232v-8.61536zm5.93549 4.03674-5.93549 2.92226-5.93549-2.92232 5.93549-2.92206zm-12.43549 7.64862v-6.81121l6 2.95349v6.81158zm13 0-6 2.95386v-6.81164l6-2.9535z"
-              />
-            </svg>
-            <span class="iui-label">
-              <div>Model Beta</div>
-              <div class="iui-description">0x0123456789</div>
-            </span>
-            <!-- Use caret-down-small.svg from @itwin/itwinui-icons package. -->
-            <svg
-              viewBox="0 0 16 16"
-              aria-hidden="true"
-              class="iui-icon"
-            >
-              <path
-                d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
-              />
-            </svg>
-          </button>
+          <nav aria-label="breadcrumbs">
```

Comment:
I am afraid that we can't guarantee that it is behaving as a breadcrumb and may do more harm than good as user would "see" several breadcrumbs. 
Few examples:
![image](https://user-images.githubusercontent.com/36186912/118435808-546c8c80-b6e8-11eb-8ce0-2dc9d8b900a7.png)
![image](https://user-images.githubusercontent.com/36186912/118435875-76660f00-b6e8-11eb-9a36-05dab614df7d.png)

Reply:
Yeah, need to think about it.. As we talked, users can pass anything under "breadcrumbs" prop which not necessarily need to be actual page breadcrumbs..

Reply:
Well, this would be applied to the `HeaderBreadcrumbs` component in React. If the user passes some other component to the `breadcrumbs` prop, it will not be applied

Reply:
But header works only with breadcrumbs components.. doesnt it? I mean, users would need to add all the classes if they pass their custom stuff...

Reply:
Ah wait.. so users can pass just array of headerbuttons. In that case, we would not apply anything, right?
So maybe it makes sense then...

Reply:
`HeaderBreadcrumbs` does one thing currently: add chevrons between `HeaderButton`s. After this change, it would also add a `nav` wrapper. So user could just pass `HeaderButton`s directly.

Reply:
yeah, in this case it makes sense..
@bentleyvk ?

Reply:
Then it is ok I guess.